lapis2/src/main/kotlin/org/genspectrum/lapis/config/SequenceFilterFields.kt

```diff
@@ -29,11 +40,19 @@ private fun mapToSequenceFilterFields(databaseMetadata: DatabaseMetadata) = when
     )
 }
 
+private fun mapToSequenceFilterFieldsFromFeatures(databaseFeature: DatabaseFeature) = when (databaseFeature.name) {
+    "sarsCoV2VariantQuery" -> "variantQuery" to SequenceFilterFieldType.VariantQuery
+    else -> throw IllegalArgumentException(
+        "Unknown feature '${databaseFeature.name}'",
```

Comment:
It might be that we will have to remove this exception, as I can imagine that not all features target filter fields, but let's change it as soon as we have to and leave it as it is for now.